diff --git a/flow/scripts/final_report.tcl b/flow/scripts/final_report.tcl
index 34d85b59..e3f61940 100644
--- a/flow/scripts/final_report.tcl
+++ b/flow/scripts/final_report.tcl
@@ -36,28 +36,24 @@ if {[info exist ::env(RCX_RULES)]} {
   # Read Spef for OpenSTA
   read_spef $::env(RESULTS_DIR)/6_final.spef
 
-  if { [info exist ::env(SKIP_IR_DROP_ANALYSIS)] && $::env(SKIP_IR_DROP_ANALYSIS) } {
-    puts "Skipped IR drop analysis"
-  } else {
-    # Static IR drop analysis
-    if {[info exist ::env(PWR_NETS_VOLTAGES)] && [string length $::env(PWR_NETS_VOLTAGES)] > 0} {
-      dict for {pwrNetName pwrNetVoltage}  {*}$::env(PWR_NETS_VOLTAGES) {
-          set_pdnsim_net_voltage -net ${pwrNetName} -voltage ${pwrNetVoltage}
-          analyze_power_grid -net ${pwrNetName} \
-              -error_file $::env(REPORTS_DIR)/${pwrNetName}.rpt
-      }
-    } else {
-      puts "IR drop analysis for power nets is skipped because PWR_NETS_VOLTAGES is undefined"
+  # Static IR drop analysis
+  if {[info exist ::env(PWR_NETS_VOLTAGES)] && [string length $::env(PWR_NETS_VOLTAGES)] > 0} {
+    dict for {pwrNetName pwrNetVoltage}  {*}$::env(PWR_NETS_VOLTAGES) {
+        set_pdnsim_net_voltage -net ${pwrNetName} -voltage ${pwrNetVoltage}
+        analyze_power_grid -net ${pwrNetName} \
+            -error_file $::env(REPORTS_DIR)/${pwrNetName}.rpt
     }
-    if {[info exist ::env(GND_NETS_VOLTAGES)] && [string length $::env(GND_NETS_VOLTAGES)] > 0} {
-      dict for {gndNetName gndNetVoltage}  {*}$::env(GND_NETS_VOLTAGES) {
-          set_pdnsim_net_voltage -net ${gndNetName} -voltage ${gndNetVoltage}
-          analyze_power_grid -net ${gndNetName} \
-              -error_file $::env(REPORTS_DIR)/${gndNetName}.rpt
-      }
-    } else {
-      puts "IR drop analysis for ground nets is skipped because GND_NETS_VOLTAGES is undefined"
+  } else {
+    puts "IR drop analysis for power nets is skipped because PWR_NETS_VOLTAGES is undefined"
+  }
+  if {[info exist ::env(GND_NETS_VOLTAGES)] && [string length $::env(GND_NETS_VOLTAGES)] > 0} {
+    dict for {gndNetName gndNetVoltage}  {*}$::env(GND_NETS_VOLTAGES) {
+        set_pdnsim_net_voltage -net ${gndNetName} -voltage ${gndNetVoltage}
+        analyze_power_grid -net ${gndNetName} \
+            -error_file $::env(REPORTS_DIR)/${gndNetName}.rpt
     }
+  } else {
+    puts "IR drop analysis for ground nets is skipped because GND_NETS_VOLTAGES is undefined"
   }
 
 } else {
