diff --git a/flow/platforms/ihp-sg13g2/lef/sg13g2_stdcell.lef b/flow/platforms/ihp-sg13g2/lef/sg13g2_stdcell.lef
index 48e8bbda..3721096c 100644
--- a/flow/platforms/ihp-sg13g2/lef/sg13g2_stdcell.lef
+++ b/flow/platforms/ihp-sg13g2/lef/sg13g2_stdcell.lef
@@ -1272,7 +1272,7 @@ MACRO sg13g2_and4_2
 END sg13g2_and4_2
 
 MACRO sg13g2_antennanp
-  CLASS CORE ANTENNACELL ;
+  CLASS CORE ;
   ORIGIN 0 0 ;
   FOREIGN sg13g2_antennanp 0 0 ;
   SIZE 1.44 BY 3.78 ;
@@ -3374,7 +3374,7 @@ MACRO sg13g2_einvn_8
 END sg13g2_einvn_8
 
 MACRO sg13g2_fill_1
-  CLASS CORE SPACER ;
+  CLASS CORE ;
   ORIGIN 0 0 ;
   FOREIGN sg13g2_fill_1 0 0 ;
   SIZE 0.48 BY 3.78 ;
@@ -3402,7 +3402,7 @@ MACRO sg13g2_fill_1
 END sg13g2_fill_1
 
 MACRO sg13g2_fill_2
-  CLASS CORE SPACER ;
+  CLASS CORE ;
   ORIGIN 0 0 ;
   FOREIGN sg13g2_fill_2 0 0 ;
   SIZE 0.96 BY 3.78 ;
@@ -3430,7 +3430,7 @@ MACRO sg13g2_fill_2
 END sg13g2_fill_2
 
 MACRO sg13g2_fill_4
-  CLASS CORE SPACER ;
+  CLASS CORE ;
   ORIGIN 0 0 ;
   FOREIGN sg13g2_fill_4 0 0 ;
   SIZE 1.92 BY 3.78 ;
@@ -3458,7 +3458,7 @@ MACRO sg13g2_fill_4
 END sg13g2_fill_4
 
 MACRO sg13g2_fill_8
-  CLASS CORE SPACER ;
+  CLASS CORE ;
   ORIGIN 0 0 ;
   FOREIGN sg13g2_fill_8 0 0 ;
   SIZE 3.84 BY 3.78 ;
